pkg/scheduler/framework/interface.go

```diff
@@ -471,41 +471,12 @@ type Framework interface {
 	// cycle is aborted.
 	RunPreFilterPlugins(ctx context.Context, state *CycleState, pod *v1.Pod) *Status
 
-	// RunFilterPlugins runs the set of configured Filter plugins for pod on
```

Comment:
Can you move the comments to the methods in Handle#PluginsRunner?

Reply:
Done